packages/data/lib/src/database/dao/people_dao.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:data/src/database/dto/people_with_image_dto.dart';
+import 'package:floor/floor.dart';
+
+@dao
+abstract class PeopleDao {
+  @Insert(onConflict: OnConflictStrategy.replace)
+  Future<List<int>> insertPeoples(List<PeopleWithImageDto> peoples);
```

Comment:
```suggestion
  Future<List<int>> insertPeople(List<PeopleWithImageDto> people);
```

<img width="746" alt="image" src="https://user-images.githubusercontent.com/64923802/197717727-bbbb13b9-413f-4628-ba08-e3a29e6cc77f.png">
<img width="751" alt="image" src="https://user-images.githubusercontent.com/64923802/197717763-d1cfc3de-70fa-41dd-8ebc-eb53755c151d.png">